docs/mcd.md

```diff
@@ -48,7 +48,7 @@ Each connection attribute is represented by a field element in the XML. The fiel
 | ----  | ------- | --------- | ----------- | ----------- |
 | name  | Unique name of the field: used in the platform, connection-normalizer and connection-builder | No | Names must be unique <br> Name is a Tableau-defined name **OR** prefixed with `v-` | If there is a Tableau-defined name for this attribute, that name must be used. See 'Connection Field Platform Integration' section below. |
 | label | Label that appears on the connection dialog for the field | No | | |
-| value-type | Dictates the default validation rule and the UI widget | No | Allowed Values: UI Widget Type <br> `string`: text box <br> `option`: drop-down <br> `boolean`: checkbox <br> `file`: file picker | In the 2020.2 release `file` is not supported. |
+| value-type | Dictates the default validation rule and the UI widget | No | Allowed Values: UI Widget Type <br> `string`: text field <br> `textbox`: text area <br>  `option`: drop-down  <br> `boolean`: checkbox <br> `file`: file picker | In the 2020.2 release `file` is not supported. |
```

Comment:
Can you add version info for `textbox` like we have for file.

Reply:
Confirmed the release date for this is 2020.4 and updated the doc.

Reply:
[](url)
![Screen Shot 2021-05-17 at 10 15 11 AM](https://user-images.githubusercontent.com/14967333/118529956-f5634380-b6f8-11eb-99c9-6443b1a6adb0.png)